python/pyspark/sql/dataframe.py

```diff
@@ -3076,15 +3080,17 @@ def transform(self, func: Callable[["DataFrame"], "DataFrame"]) -> "DataFrame":
         ...     return input_df.select([col(col_name).cast("int") for col_name in input_df.columns])
         >>> def sort_columns_asc(input_df):
         ...     return input_df.select(*sorted(input_df.columns))
-        >>> df.transform(cast_all_to_int).transform(sort_columns_asc).show()
+        >>> def mul_by(input_df, m):
+        ...     return input_df.select([(col(col_name) * m).alias(col_name) for col_name in input_df.columns])
+        >>> df.transform(cast_all_to_int).transform(sort_columns_asc).transform(mul_by, 3).show()
         +-----+---+
         |float|int|
         +-----+---+
-        |    1|  1|
-        |    2|  2|
+        |    3|  3|
+        |    6|  6|
         +-----+---+
```

Comment:
Can we keep the existing example and adding the new one to demonstrate the both usage ??

Reply:
@itholic sure - will make that change.

Reply:
@itholic done.